docs/next/package.json

```diff
@@ -26,6 +26,7 @@
     "@tailwindcss/typography": "^0.4.0",
```

Comment:
can we remove `@aws-sdk/client-s3` now?

Reply:
not yet. we still have versioned images, will update that and remove aws sdk in the next diff